vault/resource_approle_auth_backend_login.go

```diff
@@ -120,6 +120,10 @@ func approleAuthBackendLoginRead(d *schema.ResourceData, meta interface{}) error
 	log.Printf("[DEBUG] Reading token %q", d.Id())
 	resp, err := client.Auth().Token().LookupAccessor(d.Id())
 	if err != nil {
+		// If the token is not found (it has expired) we don't return an error
+		if strings.Contains(err.Error(), "invalid accessor") { // Not found
```

Comment:
Since this check occurs twice with the same string, and twice with a different string, it would be great to zip it up into a function like `isExpiredTokenErr(err error) bool`, and have its internals just check for both strings.

Reply:
Yeah, you're right. I'll merge in master and add this change!